### DISCUSSION TITLE
Build previews?

### DISCUSSION THREAD

**Discussion by awvwgk** [fortran-lang/fpm-docs#31] [General]
Do we need build previews? Building locally with mamba & sphinx seems quite straight-forward, but maybe it could be useful to have a way to create a preview to quickly get feedback.

Currently the easiest way is to deploy the branch via GitHub pages from a fork:

0. enable actions on the fork at `https://github.com/<name>/fpm-docs/actions`
1. go to `https://github.com/<name>/fpm-docs/settings/secrets/actions` and add the `DEPLOY_BRANCH` secret
2. push the branch with a matching name
3. go to `https://github.com/<name>/fpm-docs/settings/pages` and select the `gh-pages` branch for deployment
4. view preview at `https://<name>.github.io/fpm-docs`

**Comment by aslozada** [comment]
The previews are very useful for reviewing minor changes, such as  typos, or to verify that the commands embedded in the text are correct.
